internal/adapter/handlebars/helpers/getdate.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"time"
+
+	"github.com/aymerick/raymond"
+	"github.com/mickael-menu/zk/internal/util"
+	naturaldate "github.com/tj/go-naturaldate"
+)
+
+// RegisterGetDate registers the {{getdate}} template helper to use the `naturaldate` package to generate time.Time based on language strings.
+// This can be used in combination with the `date` helper to generate dates in the user's language.
+// {{date (get-date "last week") "timestamp"}}
+func RegisterGetDate(logger util.Logger) {
```

Comment:
You will need to register the helper here, otherwise it won't be available:

https://github.com/zalegrala/zk/blob/da5b7d12341016f937f35a9a9a84683af900a0e4/internal/adapter/handlebars/handlebars.go#L16

Reply:
Updated, thank you.